crates/sui-types/src/gas.rs

```diff
@@ -103,6 +103,7 @@ pub struct GasCostSummary {
     pub computation_cost: u64,
     pub storage_cost: u64,
     pub storage_rebate: u64,
+    pub non_refundable_storage_rebate: u64,
```

Comment:
Can you add doc comment on this field and stoarge_rebate field? Specifically, it's not super clear whether these two are overlapping

Reply:
Otherwise LGTM

Reply:
In my PR I actually defined `storage_rebate` to be the total rebate including both refunded and non-refundable rebates but happy to change it to use your definition. And agreed with @lxfind that we should have better comments. I would vote for more descriptive field names too.

Reply:
to be clear and make sure we are on the same page. Our decision was
storage_rebate -> the amount given back to the user
non_refundable_storage_rebate -> the small percentage kept by the system

Reply:
Ok then. I'll change my code.

Reply:
any idea on the field names? @emmazzz @lxfind @sblackshear @bmwill @tzakian